src/lib/builders/accordion/create.ts

```diff
@@ -28,10 +28,13 @@ export const createAccordion = <T extends AccordionType = 'single'>(
 	props?: CreateAccordionProps<T>
 ) => {
 	const withDefaults = { ...defaults, ...props } as CreateAccordionProps<T>;
-	const options = writable({
-		disabled: withDefaults.disabled,
-	});
 
+	const type = writable(withDefaults.type);
+	const disabled = writable(withDefaults.disabled ?? false);
+	const options = {
+		type,
+		disabled,
+	};
```

Comment:
We probably can have a wrapper for this.

e.g. `const options = withWritables(withDefaults); const {type, disabled} = options`

Withdefaults should already contain the default variable for disabled, so it is weird to have `withDefaults.disabled ?? false`